tests/test_libgsf_load.py

```diff
@@ -0,0 +1,47 @@
+from os import environ, path
+from unittest import TestCase
+
+
+class test_libgsf_load(TestCase):
```

Comment:
It would be good to convert these tests to pytest so it's consistent with the rest of the codebase

Reply:
Done